<ui version="4.0" >
 <class>dlgPlotOptions</class>
 <widget class="QDialog" name="dlgPlotOptions" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>278</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle" >
   <string>Plot Options</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>599</y>
     <width>731</width>
     <height>25</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="mcmbBLineW" >
   <property name="geometry" >
    <rect>
     <x>400</x>
     <y>100</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="editable" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="mcmbBLineStyle" >
   <property name="geometry" >
    <rect>
     <x>400</x>
     <y>70</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="mcmbBMarkerSize" >
   <property name="geometry" >
    <rect>
     <x>400</x>
     <y>160</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="editable" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>329</x>
     <y>190</y>
     <width>82</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Marker Color:</string>
   </property>
  </widget>
  <widget class="ColorButton" name="mtoolBtnColor" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>400</x>
     <y>190</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection" >
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text" >
    <string>Select Color...</string>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>339</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Line Width:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6" >
   <property name="geometry" >
    <rect>
     <x>335</x>
     <y>160</y>
     <width>82</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Marker Size:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>342</x>
     <y>70</y>
     <width>82</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Line Style:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>130</y>
     <width>82</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Marker Style:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="mcmbBMarkerStyle" >
   <property name="geometry" >
    <rect>
     <x>400</x>
     <y>130</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="mlineEditYmin" >
   <property name="geometry" >
    <rect>
     <x>202</x>
     <y>70</y>
     <width>117</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="min_label" >
   <property name="geometry" >
    <rect>
     <x>26</x>
     <y>70</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Minimum:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="mchkBoxLogX" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>150</y>
     <width>83</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Log X Axis</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mlineEditTitle" >
   <property name="geometry" >
    <rect>
     <x>77</x>
     <y>16</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Plot Title</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="mchkBoxGrid" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>180</y>
     <width>83</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Grid Lines</string>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="mlineEditYmax" >
   <property name="geometry" >
    <rect>
     <x>202</x>
     <y>96</y>
     <width>117</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="mlineEditYlabel" >
   <property name="geometry" >
    <rect>
     <x>202</x>
     <y>122</y>
     <width>117</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>y-axis</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>202</x>
     <y>33</y>
     <width>117</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Y Axis</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="mlineEditXmin" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>70</y>
     <width>116</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="plot_title_label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Plot Title:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="mchkBoxLogY" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>150</y>
     <width>83</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Log Y Axis</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mlineEditXmax" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>96</y>
     <width>116</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="mlineEditXlabel" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>122</y>
     <width>116</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>x-axis</string>
   </property>
  </widget>
  <widget class="QLabel" name="axis_text_label" >
   <property name="geometry" >
    <rect>
     <x>26</x>
     <y>122</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Label:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>33</y>
     <width>116</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>X Axis</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="max_label" >
   <property name="geometry" >
    <rect>
     <x>26</x>
     <y>96</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Maximum:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="mBtnBox" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>240</y>
     <width>431</width>
     <height>25</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="Line" name="line" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>220</y>
     <width>521</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QToolButton</extends>
   <header>colorbutton</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>mBtnBox</sender>
   <signal>accepted()</signal>
   <receiver>dlgPlotOptions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>295</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mBtnBox</sender>
   <signal>rejected()</signal>
   <receiver>dlgPlotOptions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>295</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
